plugins/ipqualityscore/plugin.spec.yaml

```diff
@@ -0,0 +1,666 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: ipqualityscore
+title: IPQS Fraud and Risk Scoring
+description: IPQS Fraud and Risk Scoring plugin provides enterprise grade fraud prevention, risk analysis, and threat detection. Analyze IP Addresses, Email Addresses, Phones, URLs and Domains to identify sophisticated bad actors and high risk behavior.
+version: 1.0.0
+supported_versions: ["1.0.0"]
+vendor: ipqs
+status: []
+tags: [ip, url, email, phone, domain]
+hub_tags:
+  use_cases: [data_enrichment,threat_detection_and_response]
+  keywords: [ip, url, email, phone, domain]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/ipqualityscore
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.ipqualityscore.com/
+enable_cache: true
+connection:
+  apiKey:
+    title: IPQS KEY
+    description: IPQualityScore API Key
+    type: credential_secret_key
+    required: true
+actions:
+  ipLookup:
+    title: IP Address Reputation
+    description: IP Address Reputation
+    input:
+      ipAddress:
```

Comment:
For every input/output would it be possible to add an example field and provide an example value?